scripts/generate_dump

```diff
@@ -863,24 +864,109 @@ save_redis() {
 }
 
 ###############################################################################
-# SAI DUMP from syncd
+# GET ROUTE table size by ASIC id and ip version
+# Globals:
+#  TIMEOUT_MIN
+#  TIMEOUT_EXIT_CODE
+# Arguments:
+#  asic id
+#  IP version
+# Returns:
+#  Status: 0 success, otherwise failure
+###############################################################################
+get_route_table_size_by_asic_id_and_ipver() {
+    local asic_id="$1"
+    local ip_ver="$2"
+    local filepath="/tmp/route_summary.txt"
+    RC=0
+
+    if [ $ip_ver = "ipv4" ]; then
+        cmd="vtysh  -n $asic_id  -c 'show ip route summary json'"
```

Comment:
-n option is not valid for a single ASIC device. Will need to check this here before adding the "-n" option

Reply:
@judyjoseph, please help to review it again.